_data/whats-new.yml

```diff
@@ -4,8 +4,44 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Thu Jun 27 16:05:11 2019
+updated: Mon Jul 8 10:58:42 2019
 entries:
+- description: Added the 'Product view page available containers' section to [Product
+    Layouts](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/product-layouts.html).
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: July 8, 2019
+  link: https://github.com/magento/devdocs/pull/4630
+- description: Clarified the Functional testing for Docker is for ece-tools and not
+    for the Magento application.
+  versions: 2.x
+  type: Technical changes
+  date: June 29, 2019
+  link: https://github.com/magento/devdocs/pull/4874
+- description: Release Notes for the Google Shopping ads Channel extension have been
+    updated to include version 2.0.0. The Upcoming Releases page has also been updated.
+  versions: 2.0 Google Shopping ads
+  type: Major update
+  date: June 28, 2019
+  link: https://github.com/magento/devdocs/pull/4850
+- description: "- Documented the `{all}` route template option for the `routes.yaml`
```

Comment:
Why is this description enclosed in " "?  In the PR, it's two bulleted items.

Reply:
The double quotes are added by the YAML formatter automatically. I assume that this done because the string starts with `-` and hyphen is a special symbol in YAML that indicates an array.

Reply:
so looking at a previous What's New entry that includes a list: https://github.com/magento/devdocs/pull/4839 -- looks like I need to add an opening sentence.

Reply:
It is up to your choice. We do not have such requirement.

Reply:
OK -- I updated it so the items would be in a list.  Thanks.